<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-933PBR5" failures="2" tests="8" name="LinkedIn Test" time="121.99" errors="0" timestamp="07 Feb 2020 15:45:11 GMT">
  <testcase classname="Tests.LoginTest" name="LoginWithInvalidPassword" time="4.78"/>
  <testcase classname="Tests.LoginTest" name="LoginWithInvalidPhoneNumber" time="1.066"/>
  <testcase classname="Tests.LoginTest" name="LoginWithValidEmailAndPassword" time="7.526"/>
  <testcase classname="Tests.LoginTest" name="LoginWithValidPhoneNumber" time="7.058"/>
  <testcase classname="Tests.RegisterTest" name="RegisterWithExistingEmail" time="0.541">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Tests.RegisterTest.RegisterWithExistingEmail(RegisterTest.java:86)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- RegisterWithExistingEmail -->
  <testcase classname="Tests.RegisterTest" name="RegisterWithInvalidEmail" time="0.555"/>
  <testcase classname="Tests.RegisterTest" name="RegisterWithLongPassword" time="1.1">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Tests.RegisterTest.RegisterWithLongPassword(RegisterTest.java:78)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- RegisterWithLongPassword -->
  <testcase classname="Tests.RegisterTest" name="RegisterWithShortPassword" time="0.615"/>
</testsuite> <!-- LinkedIn Test -->
